### PR TITLE
Seed script for mock data

### PR DESCRIPTION
**Pull request by manuelag19**

### CODE REVIEW

File: seed.js [hunk 1]
```diff
@@ -8,25 +8,23 @@
...
-//   ],
+//        },
+//      },
+//   }...],
 // }
 
-const faker = require('faker')
+const faker = require('faker');
 
 // There are usually less organizations than there are events
```
Comment (emkwak) [inline seed.js]: comments are helpful for PR reviews

File: seed.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
-// Group model in MongoDB
-// Group =
+// Org model in MongoDB
```
Comment (emkwak) [inline seed.js]: Remove before merging to master

File: seed.js [hunk 3]
```diff
@@ -73,4 +71,44 @@ var memberIds = (max) => {
     ids.push(newId);
   }
   return ids;
+}
+
+var orgEvents = (range) => {
```
Comment (emkwak) [inline seed.js]: Everything looks good. Splitting up Org / Org Events looks cleaner

File: seed.js (hunk 2 above)
Comment (manuelag19) [reply seed.js]: Thanks, new version doesn't have it.

**Pull request merged**